openfisca_dummy_country/model/model.py

```diff
@@ -134,7 +131,20 @@ def function_2011_2012(individu, period):
     def function_2013(individu, period):
         salaire_imposable = individu('salaire_imposable', period, options = [DIVIDE])
 
-        return (salaire_imposable < 500) * 300
+        return (salaire_imposable < 500) * 300.0
+
+
+class rmi(DatedVariable):
+    column = FloatCol
+    entity = Individu
+    label = u"RMI (remplacé par le RSA en 2010)"
+    definition_period = MONTH
+
+    @dated_function(datetime.date(2000, 1, 1), datetime.date(2009, 12, 31))
```

Comment:
Inconsistency: later, this is used as `start = `, and here the parameters are only indexed.